src/_includes/core-nodes/write-file-use-case.md

```diff
@@ -0,0 +1,62 @@
+## What is the Write File Node in Node-RED?
+
+The "Write File" node in Node-RED is used to write data to a file on the filesystem. It's commonly employed in flows where you need to save data or logs for later analysis or storage. In FlowFuse Cloud, the Write File node interacts with a cloud-based storage solution, leveraging AWS S3 for file storage. However, in the Node-RED instance running on edge devices using the FlowFuse device agent, this node will interact with the local file system of that device. The content to be written is specified using `msg.payload`.
```

Comment:
```suggestion
The "Write File" node in Node-RED is used to write data to a file on the filesystem. It's commonly employed in flows where you need to save data or logs for later analysis or storage. In FlowFuse Cloud, the Write File node interacts with a cloud-based storage solution, leveraging AWS S3 for file storage. However, in the Node-RED instance running on edge devices using the[ FlowFuse device agent](https://flowfuse.com/docs/device-agent/introduction/), this node will interact with the local file system of that device. The content to be written is specified using `msg.payload`.
```

Reply:
No need to add the domain when interlinking. If we do not add the domain, then the build will test which links are not valid for us.